wp-content/themes/thinkery/style.css

```diff
@@ -1485,23 +1486,55 @@ p.entry-meta {
 	margin-bottom: 0;
 }
 
+.footer-widgets div div section div div div {
+display: flex;
+justify-content: space-between;
+font-size: 1.2em;
+}
+
+.footer-widgets .widget-title {
+    text-transform: uppercase;
+	}
+
+.footer-widgets a {
+    color: #fff;
+		text-decoration: none;
+}
+
+.footer-widgets a:hover,
+.footer-widgets a:focus, {
```

Comment:
```suggestion
.footer-widgets a:focus {
```
That's the bug :)

Reply:
YAS! Code review is so much fun! Thank you!